skbeam/core/accumulators/binned_statistic.py

```diff
@@ -620,7 +620,7 @@ def __call__(self, values, statistic=None):
             The values of the selected statistic in each bin.
         """
         # check for what I believe could be a common error
-        if values.shape != self.expected_shape:
+        if tuple(values.shape) != tuple(self.expected_shape):
```

Comment:
IIUC, the issue came from passing in the shape as a list into the `expected_shape` input parameter to the `__init__`?